app/javascript/react/components/BreweryTile.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
+
 
 const BreweryTile = props => {
   return (
     <div className="brewery-tile columns small-12 medium-4 large-3">
-      <h4>{props.name}</h4>
-      <br />
+      <Link to={`/breweries/${props.id}`}>
+        <h4>{props.name}</h4>
+      </Link>
+      <img src={props.image} width = "200" />
+      <br/>
```

Comment:
alas, no `<br/>`. Nowadays there are anti-convention unless you are just playing around. Instead, add a sprinkle of CSS